tests/test_groupby_agg.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+import uuid
+
+import dask.dataframe as dd
+import pytest
+from coiled._beta import ClusterBeta as Cluster
+from dask.distributed import Client
+
+SOFTWARE = os.environ.get(
+    "COILED_SOFTWARE_NAME",
+    f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}",
+)
+
+
+@pytest.mark.parametrize("partition_size", ["10MB", "100MB", "1GB"])
+def test_quickstart(partition_size):
+    with Cluster(
+        software=SOFTWARE,
+        name="test_groupby_agg" + str(uuid.uuid4()),
+        account="dask-engineering",
+        n_workers=10,
+    ) as cluster:
```

Comment:
See the recent fixtures work. You might want to check out newer PRs like #41 for a model.

Reply:
Updated